### DISCUSSION TITLE
https://1kevinson.com/understand-how-spring-boot-architecture-is-designed/

### DISCUSSION THREAD

**Discussion by giscus** [1kevinson/BLOG-TUTORIALS#19] [Announcements]
# https://1kevinson.com/understand-how-spring-boot-architecture-is-designed/

This article provides an overview of the architecture of Spring Boot, and I offer my brief opinion on that architecture.

https://1kevinson.com/understand-how-spring-boot-architecture-is-designed/

**Comment by defofotg** [comment]
I am a little bit confused, When i read Springboot architecture, i was expecting the article to talk about annotations, the embedded webserver, the starters. But it seems like you are talking about a "spring boot project structure". From my point of view it would be more suitable as title ;)

**Reply by defofotg** [reply]
But still this is a well written article as usual!

**Reply by 1kevinson** [reply]
Yeah, this article is more about recommended spring boot project structure. But I've chosen to use the word "architecture", because most of the time when we talk about the structure of projects with Spring boot, we're talking about layered architecture 🏛️ or the repository pattern.
